gradle.properties

```diff
@@ -30,3 +30,6 @@ javaVersion=1.8
 
 docker_image_default_repo_root=apache
 docker_image_default_repo_prefix=beam_
+#gcsProject=
+gcpProject=apache-beam-build
```

Comment:
What is this? This is the cause of the Dataflow integration test failures. I don't think any of these changes should be necessary.